### PR TITLE
Add files for lab9

### PR DESCRIPTION
**Pull request by OFedorovych**

### CODE REVIEW

File: Lab2/src/main/java/ua/lviv/iot/Good.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        return weightInGrams;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public void setWeightInGrams(int weightInGrams) {
+        this.weightInGrams = weightInGrams;
+    }
```
Comment (Iggdraisil) [inline Lab2/src/main/java/ua/lviv/iot/Good.java]: getters and setters are generated by lombok

File: Lab2/src/main/java/ua/lviv/iot/Good.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+import lombok.*;
+
+
+
+@Data
+@NoArgsConstructor
+public class Good {
+	
+	private Integer id;
```
Comment (Iggdraisil) [inline Lab2/src/main/java/ua/lviv/iot/Good.java]: fix indentation

File: Lab2/src/main/java/ua/lviv/iot/controllers/GoodOrderController.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    public ResponseEntity<Good> updateGood(@RequestBody Good good) {
+        if (goodOrderService.getGood(good.getId()) != null) {
+            return new ResponseEntity<Good>(goodOrderService.updateGood(good), HttpStatus.OK);
+        } else {
+            return new ResponseEntity<Good>(HttpStatus.NOT_FOUND);
+        }
+    }
+
+    @DeleteMapping
+    public ResponseEntity<Good> deleteGood(@RequestBody Good good) {
```
Comment (Iggdraisil) [inline Lab2/src/main/java/ua/lviv/iot/controllers/GoodOrderController.java]: delete method should be called by path variable, not request body if it is possible

File: Lab2/src/main/java/ua/lviv/iot/service/GoodOrderService.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    public List<Good> getGoods() {
+        return goodsMap.values().stream().collect(Collectors.toList());
+    }
+
+    public Good getGood(Integer id) {
+        return goodsMap.get(id);
+    }
+
+    public List<Good> delGood(Good good){
```
Comment (Iggdraisil) [inline Lab2/src/main/java/ua/lviv/iot/service/GoodOrderService.java]: delete should return only deleted entity

**Review by OFedorovych — COMMENTED** [review]
corrected code according to all comments

**Pull request merged**